tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist=py27,py3
+envlist=py35,py36
```

Comment:
you probably want to update `.travis.yaml` and maybe extend pyenv to include 3.6 and 3.7?